src/web/web-service/frontend/index.html

```diff
@@ -163,27 +162,30 @@ <h4 style="margin: 18px 12px 12px 24px">OT Border Router</h4>
               <div layout="row">
                 <md-input-container flex="50">
                   <label>Network Name</label>
-                  <input required name="networkName" ng-model="thread.networkName">
+                  <input required md-maxlength="16" name="networkName" ng-model="thread.networkName">
                   <div ng-messages="threadForm.networkName.$error">
                     <div ng-message="required">This is required.</div>
+                    <div ng-message="md-maxlength">The Netowork Name must be less than 16 characters long.</div>
```

Comment:
yes, in Thread spec v1.1 (page 8-26), there is the interpretation about the network name.